modules/build/src/main/scala/scala/build/preprocessing/mdsandbox/MdRunner.scala

```diff
@@ -0,0 +1,30 @@
+package scala.build.preprocessing.mdsandbox
+
+import scala.build.Inputs
+
+object MdRunner {
```

Comment:
Hmm... `MdRunner` doesn't seem like a fortunate name, as far as I understand the code, this seems to be just a handler for wrapping the Scala snippets from markdown. So maybe:
```suggestion
object MarkdownScalaSnippetRunner {
```
This also makes it clearer, that we will need the equivalent of this class to handle Java snippets in the future.